src/packages/settings/dashboards/telemetry/dashboard-telemetry.element.ts

```diff
@@ -113,26 +105,16 @@ export class UmbDashboardTelemetryElement extends UmbLitElement {
 	render() {
 		return html`
 			<uui-box class="uui-text">
-				<h1 class="uui-h2">Consent for telemetry data</h1>
+				<h1 class="uui-h2"><umb-localize key="analytics_consentForAnalytics"></umb-localize></h1>
```

Comment:
Ahh, I think it could work if you include a fallback text here, so the tester can see it:

```suggestion
				<h1 class="uui-h2"><umb-localize key="analytics_consentForAnalytics">Consent for telemetry data</umb-localize></h1>
```